adhocracy4/images/mixins.py

```diff
@@ -0,0 +1,59 @@
+from django import forms
+from django.utils.translation import gettext_lazy as _
+
+
+class ImageMetadataMixin(forms.ModelForm):
+    def __init__(self, *args, image="image", tile_image="tile_image", **kwargs):
+        self.image = image
+        self.tile_image = tile_image
+        super().__init__(*args, **kwargs)
+
+        if not self.instance.image:
+            self.initial["image_copyright"] = ""
+            self.initial["image_alt_text"] = ""
+        if self.instance.image and not self.instance.image_copyright:
+            self.initial["image_copyright"] = _("Copyright of project owner")
```

Comment:
This is not the final wording and it doens't actually work like a default, it just adds it when the image is saved rather then adding it to the old projects, I assume I am doing this wrong?